core/src/impl/Kokkos_HBWSpace.cpp

```diff
@@ -192,6 +192,29 @@ SharedAllocationRecord<Kokkos::Experimental::HBWSpace,
                      SharedAllocationRecord<void, void>::m_alloc_size);
 }
 
+// TODO un-copy-pasta this when we un-copy-pasta the rest of HBWSpace
```

Comment:
When do you expect to get to it?  Please add a comment that the duplicated code lives in `impl/Kokkos_HostSpace.cpp`?

Reply:
This is a much bigger project, since the *entire* 
`HBWSpace` implementation looks to be copy-pasta. We should open a separate issue. But yeah, the comment is a really good idea, thanks.